_datalad_build_support/setup.py

```diff
@@ -13,13 +13,11 @@
 import sys
 
 
-from distutils.core import Command
-from distutils.errors import DistutilsOptionError
-from distutils.version import LooseVersion
 from genericpath import exists
 from os import linesep, makedirs
 from os.path import dirname, join as opj, sep as pathsep, splitext
-from setuptools import findall, find_packages, setup
+from setuptools import Command, DistutilsOptionError, findall, find_packages, setup
```

Comment:
Please follow the format yielded by `python -m isort -m3 -fgw 2 -tc`, apart from other aspects, one import per line.

Reply:
Done.